javascript/pull-view.js

```diff
@@ -33,6 +33,11 @@
         this.$el.addClass("thumbsup");
       }
 
+      this.wip = (this.model.get('title').indexOf('WIP') >= 0);
```

Comment:
From a quick glance, it seems that the only other one needed is "DO NOT MERGE". I could add that in.